code/game/objects/items/dildo

```diff
@@ -0,0 +1,51 @@
+//Coded by Nexendia~
+
+/obj/item/dragon
+	name = "Bad Dragon"
+	desc = "You really shouldn't see this.. but if you do... Huzzah.. you found a bug.. please scream at Nexendia!"
+	icon = 'icons/obj/dicks.dmi'
+	icon_state = null
+	force = 5
+	w_class = 1
+	throwforce = 5
+	hitsound = 'sound/items/squishy.ogg'
+	attack_verb = list("slapped")
+
+/obj/item/dragon/suicide_act(mob/user)
+	user.visible_message(pick("<span class='suicide'>[user] is shoving [src.name] down \his throat! It looks like they're trying to commit suicide.</span>"))
+	return(BRUTELOSS)
+
+/obj/item/dragon/sea
+	name = "Sea Dragon Dildo"
+	desc = "A Sea Dragon Dildo, Why this is on a SpaceStation we will never know..."
+	icon_state = "seadragon"
+	force = 5
```

Comment:
No need to redefine force and throwforce since parent already has them to the same value